README.md

```diff
@@ -1,170 +1,107 @@
-# Android Base Project
+# BigBang
 
-This is the Xmartlabs' Android base project. Remember to follow [the style guide]
-(https://github.com/xmartlabs/Android-Style-Guide) when working on projects, and also when contributing here.
-An example usage of this project lays in [the demo branch]
-(https://github.com/xmartlabs/bigbang/tree/demo).
+[![CircleCI](https://circleci.com/gh/xmartlabs/bigbang.svg?style=svg)](https://circleci.com/gh/xmartlabs/Android-Base-Project)
+[![codebeat badge](https://codebeat.co/badges/af8770f0-d2bf-47d1-a504-6dee56b99312)](https://codebeat.co/projects/github-com-xmartlabs-android-base-project-master)
+[![Core](https://api.bintray.com/packages/xmartlabs/Android-Base-Project/Core/images/download.svg)](https://bintray.com/xmartlabs/Android-Base-Project/Core/_latestVersion)
+[![DbFlow](https://api.bintray.com/packages/xmartlabs/Android-Base-Project/DbFlow/images/download.svg)](https://bintray.com/xmartlabs/Android-Base-Project/DbFlow/_latestVersion)
+[![Retrofit](https://api.bintray.com/packages/xmartlabs/Android-Base-Project/Retrofit/images/download.svg)](https://bintray.com/xmartlabs/Android-Base-Project/Retrofit/_latestVersion)
+[![Ui](https://api.bintray.com/packages/xmartlabs/Android-Base-Project/Ui/images/download.svg)](https://bintray.com/xmartlabs/Android-Base-Project/Ui/_latestVersion)
 
-## Assets
 
-The configuration assets are in a [Google Drive folder](PUT LINK).
-To access them, ask via email.
+This is [XMARTLABS](https://xmartlabs.com) Android base project. Architecturally composed of a set of libraries for building modern and scalable applications from the ground up.
```

Comment:
I checked https://github.com/xmartlabs/Eureka for this. Should we change this not to be uppercase anyway?